machine.go

```diff
@@ -337,42 +398,52 @@ func (m *Machine) kernelRelease() (string, error) {
 	return "", fmt.Errorf("No kernel found")
 }
 
-func (m *Machine) writerKernelModules(w *writerhelper.WriterHelper) error {
-	kernelRelease, err := m.kernelRelease()
+func hostKernelPath() (string, string, error) {
```

Comment:
This and hostKernelRelease()  should a helper function in the kvm backend right? for UML they logic wouldn't apply

Reply:
my thinking here was to keep these helpers global (to fakemachine) since most other "backends" would use the host's kernel & modules, uml is an exception.

For instance (i am speculating here) a virtualbox backend would also use these helpers.

Reply:
We can burn that bridge when we get to it and factor things out in a utility later on. But for now it's for sure odd that machine.go has backend specific helpers ;)